src/components/ConversationList.js

```diff
@@ -114,19 +114,14 @@ class ConversationList extends PureComponent {
     } = this.props;
 
     const user = client.getCurrentUser();
-    let headerImage;
-    if(user && user.profileImageUrl) {
-      headerImage = user.profileImageUrl;
-    } else {
-      headerImage = IMAGES.AVTAR;
-    }
 
     list = list.map(conversation => modifyConversation(conversation));
 
     return (
       <div id="ch_recent_window" className="ch-recent-window">
         <Header 
-          profileImageUrl={headerImage}
+          profileImageUrl={user && user.profileImageUrl}
+          profileImageAlt={user && user.displayName}
```

Comment:
`profileImageAlt` should be renamed as `initials`.

Reply:
Done!